Scripts/UpdateInsightsTableCreationSettings.ps1

```diff
@@ -0,0 +1,58 @@
+<# 
+.SYNOPSIS 
+Script that enables/disables per process/queue table creation for UiPath Insights.
+#>
+param(
+    [Parameter(Mandatory=$true)][string]$server # Orchestrator endpoint
+    ,[Parameter(Mandatory=$true)][string]$tenancyName # Tenant name
+    ,[Parameter(Mandatory=$true)][string]$user # Tenant user name
+    ,[secureString]$password = $(Read-Host -Prompt "Enter password" -AsSecureString) # Tenant user password (prompted)
+    ,[boolean]$perProcessTablesEnabled = $true # Boolean to enable/disable process tables
+    ,[boolean]$perQueueTablesEnabled = $true # Boolean to enable/disable queue tables
+)
+
+$credentials = New-Object System.Management.Automation.PSCredential `
+    -ArgumentList $user, $password
+
+# Base headers
+$headers = @{
+    'Accept' = 'application/json'
+    'Content-Type' = 'application/json'
```

Comment:
Do we need a `;` as a separator for each entry?

Reply:
Nope, dictionaries in powershell appear to be separated by a new line.